internal/js/summary.js

```diff
@@ -270,6 +274,64 @@ class ReportBuilder {
 		return this;
 	}
 
+	/**
+	 * Computes the global maximum name width across all metrics in all sections.
+	 * This ensures consistent colon alignment across the entire report.
+	 *
+	 * @param {Report} report - The complete report object
+	 * @returns {void}
+	 */
+	_computeGlobalMaxNameWidth(report) {
+		const allMetrics = {};
+		const renderContext = this.renderContext.indentedContext(1);
+
+		// Collect metrics from total results
+		if (report.metrics) {
+			Object.entries(report.metrics).forEach(([_, sectionMetrics]) => {
+				Object.assign(allMetrics, sectionMetrics);
+			});
+		}
+
+		// Collect metrics from groups
+		if (report.groups) {
+			this._collectMetricsFromGroups(report.groups, allMetrics);
+		}
+
+		// Collect metrics from scenarios
+		if (report.scenarios) {
+			this._collectMetricsFromGroups(report.scenarios, allMetrics);
+		}
+
+		// Compute the global max name width
+		let maxNameWidth = 0;
+		Object.keys(allMetrics).forEach((name) => {
+			const displayName = renderContext.indent(renderMetricDisplayName(name));
```

Comment:
Do we really need to use `renderContext` here? In my opinion it's confusing because this code is treating all the metrics (total results, groups, scenarios, etc) as the same, but each level will be later indented differently.

As the purpose is to calculate the longest metric name, I suspect we could just omit the indentation for this calculus.